addon/webextension/selector/ui.js

```diff
@@ -841,9 +841,15 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       img.src = dataUrl;
       iframe.document().querySelector(".preview-image").appendChild(img);
       if (showCropWarning && !(isDownloadOnly())) {
-        let imageCroppedEl = makeEl("DIV");
-        imageCroppedEl.id = "imageCroppedWarning";
-        imageCroppedEl.textContent = browser.i18n.getMessage("imageCroppedWarning", buildSettings.maxImageHeight);
+        let imageCroppedEl = makeEl("table", "image-cropped-warning");
```

Comment:
Looks like you've left the 'image-cropped-warning' class on this table, and the other table has the 'download-only' class applied to it. Any reason a 'notice' class couldn't be applied to both?